zapcore/console_encoder.go

```diff
@@ -103,52 +102,52 @@ func (c consoleEncoder) EncodeEntry(ent Entry, fields []Field) ([]byte, error) {
 	// If there's no stacktrace key, honor that; this allows users to force
 	// single-line output.
 	if ent.Stack != "" && c.StacktraceKey != "" {
-		line.WriteByte('\n')
-		line.WriteString(ent.Stack)
+		line.AppendByte('\n')
```

Comment:
Unfortunately, this triggers a variety of irritating `go vet` checks that attempt to enforce compliance with some interfaces in the standard lib.